awx/playbooks/project_update.yml

```diff
@@ -212,10 +214,9 @@
         - name: End play due to disabled content sync
           ansible.builtin.meta: end_play
 
-    - name: Fetch galaxy roles from requirements.(yml/yaml)
+    - name: Fetch galaxy roles from roles/requirements.(yml/yaml)
       ansible.builtin.command: >
         ansible-galaxy role install -r {{ item }}
-        --roles-path {{ projects_root }}/.__awx_cache/{{ local_path }}/stage/requirements_roles
```

Comment:
Here's how I'd do the puzzle here:
 - if you're taking out `--roles-path` here, fine, counter-balance by adding `ANSIBLE_ROLES_PATH` to the shared environment blob, which is `additional_collections_env`, and adding to to `environment:` for this task
 - when you do that, obviously it's not just collections paths you're setting (which is totally fine), so just rename it to something more general, as this defines paths for collections/roles/tmp